### PR TITLE
feat: Add glasses icon

### PR DESCRIPTION
**Pull request by ahtohbi4**
Icon glasses was added

Icon glasses

### CODE REVIEW

**Review by locness3 — COMMENTED** [review]
Not a fan of this design, personnally.

**Comment by ericfennis** [comment]
Maybe try one version for comparison with straight feet?

**Comment by mittalyashu** [comment]
The naming might be bit confusing.

Suggestions:
1. Sun glasses
2. Spectacles

**Comment by locness3** [comment]
Why would it be?

**Comment by mittalyashu** [comment]
@locness3 the keyword "glass" can itself define lots of things.

CleanShot 2021-01-16 at 21 36 31@2x

_you can mark this comment as off-topic, if you want_

**Comment by locness3** [comment]
Yup but glasses (plural) generally means what you wear, and I wouldn't imagine to find an icon with multiple drkining glasses.

**Comment by ericfennis** [comment]
What do you think of longer feet? 

image

File: icons/glasses.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="6" cy="14" r="4" />
+  <circle cx="18" cy="14" r="4" />
+  <path d="M14 14C14 12.9 13.1 12 12 12C10.9 12 10 12.9 10 14" />
+  <path d="M2.5 12L4.9 7.21115C5.6 5.9 6.9 5 8.5 5H9" />
+  <path d="M21.5 12L19.1 7.21115C18.4 5.9 17 5 15.5 5H15" />
```
Comment (ericfennis) [inline icons/glasses.svg:16]:
```suggestion
  <circle cx="6" cy="15" r="4" />
  <circle cx="18" cy="15" r="4" />
  <path d="M14 15a2 2 0 00-2-2 2 2 0 00-2 2" />
  <path d="M2.5 13L5 7c.7-1.3 1.4-2 3-2" />
  <path d="M21.5 13L19 7c-.7-1.3-1.5-2-3-2" />
```

**Pull request merged**